internal/services/mssqlmanagedinstance/mssql_managed_instance_resource_test.go

```diff
@@ -1686,3 +1782,75 @@ resource "azurerm_mssql_managed_instance" "test" {
 }
 `, r.template(data, data.Locations.Primary), data.RandomInteger)
 }
+
+func (r MsSqlManagedInstanceResource) subnetUpdate(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%[1]s
+
+provider "azurerm" {
+  features {
+    resource_group {
+      /* Due to the creation of unmanaged Microsoft.Network/networkIntentPolicies in this service,
+      prevent_deletion_if_contains_resources has been added here to allow the test resources to be
+       deleted until this can be properly investigated
+      */
+      prevent_deletion_if_contains_resources = false
+    }
+  }
+}
+
+
+resource "azurerm_subnet" "test2" {
+  name                 = "subnet2-%[2]d"
```

Comment:
could we prefix this resource name with `acctest`?

```suggestion
  name                 = "acctestsubnet2-%[2]d"
```

Reply:
certainly